### PR TITLE
RackConnectV3 compatibility

### PR DESCRIPTION
**Pull request by nazgul5**
Clone of #69 PR with backward compatibility saved

### CODE REVIEW

**Comment by martinb3** [comment]
@nazgul5 does the change to servers.bootstrap/servers.new and the changed servicenet default constitute a breaking change?

**Comment by nazgul5** [comment]
Hi @martinb3

I've tried to avoid any possible breaking changes.

- servers.bootstrap => servers.new change just removes requirement for public ip address provisioning hard-coded into fog/rackspace driver
- config.servicenet=true default, with introduction of connect_public_net=true, keeps old logic intact when determining ip address to connect

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ cache: bundler
 sudo: false
 
 rvm:
+  - 2.2
```
Comment (tas50) [inline .travis.yml]: it's best to use 2.2.7 here so you get the latest vs. the rvm alias for 2.2

Comment (nazgul5) [reply .travis.yml]: @tas50 thanks, updated

**Pull request closed without merging**